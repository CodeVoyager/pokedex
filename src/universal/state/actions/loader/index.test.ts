import { startLoadingAction, stopLoadingAction } from '.';

describe('State::Actions', () => {
  describe('Loader', () => {
    describe('startLoadingAction creator', () => {
      test('should return correct action', () => {
        expect(startLoadingAction()).toMatchSnapshot();
      });
    });
    describe('stopLoadingAction creator', () => {
      test('should return correct action', () => {
        expect(stopLoadingAction()).toMatchSnapshot();
      });
    });
  });
});
