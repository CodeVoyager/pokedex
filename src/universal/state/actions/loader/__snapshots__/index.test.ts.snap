// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`State::Actions Loader startLoadingAction creator should return correct action 1`] = `
Object {
  "type": "START_LOADING",
}
`;

exports[`State::Actions Loader stopLoadingAction creator should return correct action 1`] = `
Object {
  "type": "STOP_LOADING",
}
`;
